lib/matplotlib/axes/_axes.py

```diff
@@ -1091,55 +1091,73 @@ def eventplot(self, positions, orientation='horizontal', lineoffsets=1,
                   linelengths=1, linewidths=None, colors=None,
                   linestyles='solid', **kwargs):
         """
-        Plot identical parallel lines at specific positions.
+        Plot identical parallel lines at the given positions.
 
-        Plot parallel lines at the given positions.  positions should be a 1D
-        or 2D array-like object, with each row corresponding to a row or column
-        of lines.
+        *positions* should be a 1D or 2D array-like object, with each row
+        corresponding to a row or column of lines.
 
         This type of plot is commonly used in neuroscience for representing
-        neural events, where it is commonly called a spike raster, dot raster,
+        neural events, where it is usually called a spike raster, dot raster,
         or raster plot.
 
         However, it is useful in any situation where you wish to show the
         timing or position of multiple sets of discrete events, such as the
         arrival times of people to a business on each day of the month or the
         date of hurricanes each year of the last century.
 
-        *orientation* : [ 'horizontal' | 'vertical' ]
-          'horizontal' : the lines will be vertical and arranged in rows
-          'vertical' : lines will be horizontal and arranged in columns
+        Parameters
+        ----------
+        positions : 1D or 2D array-like object.
+            Each value is an event. If *positions* is a 2D array-like, each
+            row corresponds to a row or a column of lines (depending on the
+            *orientation* parameter).
 
-        *lineoffsets* :
-          A float or array-like containing floats.
+        orientation : str [ 'horizontal' (default) | 'vertical' ], optional.
+            'horizontal' : the lines will be vertical and arranged in rows.
```

Comment:
This is confusing because it says horizontal is vertical and vertical is horizontal. I would swap the order of the descriptions so that things that are same come next to each otherm i.e., `'horizontal': the lines are arranged horizontally in rows, and are vertical.`

Also, these are not bullet points or separate paragraphs, so keep that in mind; both lines will appear as one paragraph in the rendered docs.